src/main/webapp/resources/js/apis/activities/activities.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+/**
+ * @file API for handling activities
+ */
+
+const BASE_URL = `/ajax/activities`;
+
+/**
+ * Get a page of activities for a project
+ *
+ * @param {number} projectId - current project identifier
+ * @param {number} page - page of activities requested
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export function getProjectActivities({ projectId, page = 0 }) {
+  return axios
```

Comment:
We should also catch an error if there was an issue getting the project activities. This would need to be updated on the server side and UI side as well

Reply:
Updated in c23a00b